JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeViewModel.kt

```diff
@@ -37,6 +37,8 @@ import java.util.UUID
  */
 data class HomeUiState(
     val posts: List<Post> = emptyList(),
+    val selectedPostId: String? = null, // TODO back selectedPostId in a SavedStateHandle
```

Comment:
I'd expect this to expose the actual `Post` and not the id, it's better if the UI doesn't need to figure this out